Added team page, tweaked some styling

claynaut opened the pull request:
**Changes**
- Added team page with placeholder images and proper captions
- Moved social media links out of footer and into the team page

**Misc. Changes**
- Added more theme variables for fonts
- Added a font face to use Montserrat bold (used in team page)

r-jo reviewed (changes requested):
- Either reduce page height of team section, or vertically center it.
- Add Committee list place holder
- Reduce picture size and picture/text margins for team page on mobile so you can fit two columns

claynaut commented:
**Additional Changes**
- Reduced page height of team section
- Added committee list placeholders
- Reduced picture sizes for team page so two columns fit on mobile
- Redid wrapper for images on past winners section (made into a grid) so two columns could fit on a smaller screen size (mobile still shows one column but enlarged)
- Modified pictures for winners to fit into polaroid
- Inserted pictures and added descriptions for winners
@r-jo
